packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -152,18 +152,17 @@ class InteractiveViewer extends StatefulWidget {
   /// Defaults to [Clip.hardEdge].
   final Clip clipBehavior;
 
-  /// If true, panning is only allowed in the direction of the horizontal axis
-  /// or the vertical axis.
+  /// When set to [PanAxis.aligned] panning is only allowed in the horizontal
+  /// axis or the vertical axis, diagonal panning not allowed.
   ///
-  /// In other words, when this is true, diagonal panning is not allowed. A
-  /// single gesture begun along one axis cannot also cause panning along the
-  /// other axis without stopping and beginning a new gesture. This is a common
-  /// pattern in tables where data is displayed in columns and rows.
+  /// When set to [PanAxis.vertical] or [PanAxis.horizontal] panning is only
+  /// allowed in the specified axis. For example, if set to [PanAxis.vertical]
+  /// if you try to pan in the horizontal axis the InteractiveViewer will not
+  /// pan. And if you try to pan in diagonal directions the InteractiveViewer will
```

Comment:
Can you change this to not use the word "you"?  See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require.

Otherwise the general idea makes sense to me :+1: